### ISSUE TITLE
Rewarded ad close button is greyed out and not closable after 5 seconds.

### ISSUE THREAD

**Issue by kuoliangkwong** [googleads/googleads-mobile-unity#2122]
### [REQUIRED] Step 1: Describe your environment

  * Unity version: 2021.3.2f1
  * Google Mobile Ads Unity plugin version: 7.0.0
  * Platform: iOS, Android
  * Platform OS version: Android 11, Android 12, iOS 15
  * Any specific devices issue occurs on: n/a
  * Mediation ad networks used, and their versions: n/a

#### Steps to reproduce:

What happened? How can we make the problem occur?
Rewarded ad close button is not enabled and the ad is not closable after 5 seconds. This issue causes Google Play rejecting our app update. Can be reproduced by using HelloWorld sample project in this plugin, then build a live app with live Android keystore, live admob ad unit ID and live admob app ID. Not able to reproduce with Test Ad Unit ID because it only shows total 5 seconds of video ad.

**Comment by NVentimiglia** [comment]
Hi @kuoliangkwong,

Since this is not replicated with Test Ad Units, I will need more details. Can you please submit a [help center report ](https://support.google.com/admob/thread/new). Include this github issue as well as your publisher id and ad unit id. This will help me escalate the issue.

**Comment by kuoliangkwong** [comment]
Hi @NVentimiglia, I have contacted admob support (same as the link attached in your comment) and it seems like it was fixed about 1-2 weeks ago. Suspected it was server issue as it was fixed without us making any change from our end.

**Comment by sjvc** [comment]
Same here. Ad close button is always disabled. I'm using Unity 2021.3.6f1 and Google Mobile Ads Unity plugin 7.1.0 with iOS 15 (iPhone SE)

**Comment by KishoreKumar21** [comment]
Same issue here, Google has rejected my app for the exact same reason. @kuoliangkwong has the issue been fixed now?
@NVentimiglia could you please provide an update on this? Our planned release is getting postponed because of this and is causing lot of delays and severe impact.

**Comment by kuoliangkwong** [comment]
@KishoreKumar21, 1 week after my app got rejected, the button was working. But it's not working again recently. I have contacted admob support and they said they will get back to me. I would advise y'all to contact admob support so they are alerted about this issue.

**Comment by KishoreKumar21** [comment]
@kuoliangkwong I have posted the same in google groups for admob: https://groups.google.com/g/google-admob-ads-sdk/c/hmx4mWfuDQA . You too please do the same if you haven't already. They have asked me to get back with Charles log for the same. I'm worried about testing with real ads in the game, but going to try it this weekend. Let us know when you get a resolution in case you do get.

**Comment by kuoliangkwong** [comment]
@KishoreKumar21, I have sent a couple of Charles logs but they can't find what they are looking for. For some reasons, the things they were looking for just did not show up in Charles logs. Plus, they are not able to compile an APK from the sample project I've sent, though I have put all the instructions to compile a LIVE APK to reproduce the issue with LIVE ads. They insisted to test on Test Ad Unit from their end, which is not reproducible since Test Ads only last for 5 seconds total. I ended up sending them an APK that using LIVE ads and still waiting for their response.

**Comment by KishoreKumar21** [comment]
@kuoliangkwong They did request ad request url from the Charles logs. I'm not able to send them since I have been put under limited ad serving for no reason. I did not even test with real ads. Could it be that since the google reviews the app with real ads, the admob considers it as invalid clicks? Also, one thing I noticed is I found this issue even on a game "Knife Hit" which has over 100 mil downloads, the rewarded ads are just not closable after 5 secs. Surely. there is an issue from their end, but not sure when we would get a resolution on this.
@NVentimiglia Are you still looking at this issue?

**Comment by kuoliangkwong** [comment]
Yes, we have been testing different games from Play Store, and most of them have the same issue. I'm surprised not many have filed this issue.

**Comment by KishoreKumar21** [comment]
@kuoliangkwong any luck on fixing the issue?

**Comment by doyundev** [comment]
The issue isn't fixed latest version..
Admob for Unity : v7.1.0
Rewarded Ads not closable from the beginning. (AOS, iOS Both)
has anyone solved the issue?

**Comment by thsbrown** [comment]
Having the same issue on my end 🤔.

**Comment by KishoreKumar21** [comment]
@kuoliangkwong were you able to resolve the issue?

**Comment by kuoliangkwong** [comment]
Nope. Admob support has yet to reply. I've just sent another email to follow up.

**Comment by kuoliangkwong** [comment]
@NVentimiglia, is there any ETA for this fix? I tried contacting admob support team but they said they will look into it (since the day I raised this issue) and have yet received any reply from them. Would like to follow up with you on this as this is affecting our app release schedule due to Google Play console team keeps rejecting our app update for this reason.

**Comment by madTeddy** [comment]
I have the same issue and Google Play Support has it too (they rejected my app update because of this).

**Comment by NVentimiglia** [comment]
Hi everyone,

This issue is still a priority, and we are still looking into this issue.

Thank you for your patience.

**Comment by madTeddy** [comment]
> Hi everyone,
> 
> This issue is still a priority, and we are still looking into this issue.
> 
> Thank you for your patience.

I see. Then I guess everyone will have to leave Admob, since you are still trying to find the problem, while Google Play, in the meantime, does not allow updates with this problem👍.

**Comment by HaydenCampbell** [comment]
Same problem here on Android 12. Does anyone know if this is just affecting the later versions of Android such as 11, 12 and 13? I don't seem to have a problem on earlier versions of Android.

**Comment by madTeddy** [comment]
Ok. Atleast now this problem is gone for me. Maybe they fixed it.

**Comment by HaydenCampbell** [comment]
I ended up removing rewarded ads from all my production apps for now. Not only were users unable to close the ads but the OnUserEarnedReward listener wasn't being called too and some cases the user cannot back click out of the ad, so they had to force close the app, it was causing us major problems.

**Comment by Cuxy** [comment]
I have contacted Admob Support and got the following answer: 

Since you mentioned that the rewarded app can be skipped only after the countdown is done, then it is working as intended. If you want only skippable ads, you can implement rewarded interstitial ad formats instead.

**Comment by rudolfolah** [comment]
Are the code samples on this page incorrect? https://developers.google.com/admob/unity/rewarded-interstitial

I have the rewarded interstitial loading and displaying and the close button appears however only the `OnAdDidPresentFullScreenContent` callback is called.

**Comment by kuoliangkwong** [comment]
Hey guys, just to update from my end after contacting Admob team. It seems like they were testing a new feature where the rewarded ad has full non-skippable ad. However, the 5s closeable button will still be enabled for those apps that fall under family program or targeting for children. There was a missing scope where family program app were not taken into account at Admob's end and it is fixed now. Cheers~

**Comment by rudolfolah** [comment]
Good to know! My problem was fixed when I tested on an Android device
instead of in the Unity simulator

**Comment by LordChunder** [comment]
Same issue. With release devices. Is  it a problem with raycasting the touch?

**Comment by MichalKlacko** [comment]
Hi,
i'm having the same issue. All ad formats runs fine but rewarded is not skippable before reward is granted. 
I'm testing build on real iOS device with real ads (in test mode).
X Button with "XY Seconds remaining" countdown is shown but when I click it nothing happens until it counts to 0 and grant reward, only after then the X button works.

Is this intended behaviour? 

Thanks
